lib/bundle/cask_dumper.rb

```diff
@@ -41,8 +41,14 @@ def dump(casks_required_by_formulae, describe: false)
     def dump_cask(cask, describe:)
       description = "# #{cask.desc}\n" if describe && cask.desc.present?
       config = if cask.config.present? && cask.config.explicit.present?
-        cask.config.explicit.map do |k, v|
-          "#{k}: \"#{v.sub(/^#{ENV['HOME']}/, "~")}\""
+        # TODO: replace this logic with an actual call in Library/Homebrew/cask/
+        cask.config.explicit.map do |key, value|
+          if key == "language"
+            key = "languages"
+            value = value.split(",")
+          end
+
+          "#{key}: \"#{value.sub(/^#{ENV['HOME']}/, "~")}\""
```

Comment:
@EricFromCanada Would you be able to take a look at that? CC @Homebrew/cask for help either doing this or a pointer to what the best methods would be to call to avoid this duplication here.

Reply:
Something like this? Homebrew/brew#10462